src/SFA.DAS.DATA.MANAGEMENT/Tables/AsDataPL_EI_IncentiveApplicationApprenticeship.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [AsData_PL].[EI_IncentiveApplicationApprenticeship]
+(
+	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY,
+	[IncentiveApplicationId] UNIQUEIDENTIFIER NOT NULL,
+	[ApprenticeshipId] BIGINT NOT NULL,
+	[ULN] BIGINT NOT NULL,
```

Comment:
ULN is in the excluded column list?

Reply:
Thanks @arvinthseran . This is now removed.